src/components/Cluster/ClusterDetail/Ingress/InstallIngressButton.tsx

```diff
@@ -77,11 +77,11 @@ const InstallIngressButton: React.FC<IInstallIngressButtonProps> = ({
     tryToLoadApps();
   }, [dispatch, clusterID]);
 
-  const installIngressController = async () => {
+  const installIngressController = () => {
     try {
       setIsInstalling(true);
 
-      await dispatch(installApp(appToInstall, clusterID));
```

Comment:
A consequence of the new approach, no more thunk is returned here, it is not returned by the callAPIMiddleware.. maybe it could be... but I'm not sure if we wanted that. This enforces something we wanted originally.

But that means this dispatch and the next are fired at pretty much the same moment, which isn't ideal in my mind.

In this case it means we might load cluster apps so fast that we don't see the newly installed app in the list yet.

<img width="378" alt="Screenshot 2020-06-26 at 1 31 02 AM" src="https://user-images.githubusercontent.com/455309/85769614-bc228d80-b74c-11ea-9c30-3afcc7b8d413.png">

Reply:
Before you would 100% for sure see:

```
INSTALL_APP_REQUEST
INSTALL_APP_SUCCESS
LOAD_CLUSTER_APPS_REQUEST
LOAD_CLUSTER_APPS_SUCCESS
```

Reply:
I think we can move the call to the app reloading action creator to the app install action creator, since we're really never going to install an app and not fetch the latest app list immediately after.

We can even have a common configuration argument to these type of creational/mutational action creators, whether it should refresh the list after executing the creation/mutation. I think this is a good step in the direction of simplifying the data fetching part of our view components.